styles.css

```
@@ -132,40 +134,49 @@
     background-color: #494949;
 }
 
-.slider > .range {
+.slider>.range {
     position: absolute;
     z-index: 2;
     left: 25%;
     right: 25%;
     top: 0;
     bottom: 0;
     border-radius: 5px;
-    background-color: #1ed760;
+    background-color: #fff;
 }
 
-.slider > .range:hover {
+.snippy:hover .slider>.range {
     background-color: #1ed760;
 }
 
-.slider > .thumb {
+.slider>.thumb {
     position: absolute;
     z-index: 3;
     width: 12px;
     height: 12px;
     border-radius: 50%;
     background-color: #fff;
-    box-shadow: 0 0 0 0 rgba(98,0,238,.1);
+    box-shadow: 0 0 0 0 rgba(98, 0, 238, .1);
     transition: box-shadow .3s ease-in-out;
 }
 
-.slider > .thumb.left {
+.slider>.thumb.left {
     left: 25%;
     transform: translate(-6px, -3px);
+    display: none;
 }
 
-.slider > .thumb.right {
+.slider>.thumb.right {
     right: 25%;
     transform: translate(6px, -3px);
+    display: none;
+}
+
+.snippy:hover .slider>.thumb.left,
+.snippy:hover .slider>.thumb.right,
+.snippy:active .slider>.thumb.left,
+.snippy:active .slider>.thumb.right {
+    display: block;
```

@geraldopmj : Glorious! Tested locally and looks good. 💯  Mind making these small changes? 🙏 

Like potifys, the thumbs should really only show up when a user hovers over the actual slider or at least it's container, which in this case it's the `.middle` class. I think it might be better renamed as `slider-wrapper` or `slider-container`. Anything along those lines is fine if you have something else in mind.

```suggestion
.slider-container:hover .slider>.thumb.left,
.slider-container:hover .slider>.thumb.right,
.slider-container:active .slider>.thumb.left,
.slider-container:active .slider>.thumb.right {
    display: block;
```

2. With the above change then the class name would also have to be updated: https://github.com/cdrani/chorus/blob/9f50b405b71a077f0b82918d2b5cb92623a420f9/components/slider.js#L7


Thanks again, mate!

---

Also thanks for the prettier changes! I do plan on integrating it into the project later on for everyone, but for now I just want to focus on features, bug fixes, and ui adjustments.